scripts/olm-build-bundle-image.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SCRIPTS_DIR=$DIR
+ROOT_DIR=$DIR/..
+BUILD_DATE=$(date +%Y-%m-%dT%H:%M)
+QUIET=${QUIET:-"false"}
+DEFAULT_DOCKER_REPOSITORY="amazon/aws-controllers-k8s"
+DOCKER_REPOSITORY=${DOCKER_REPOSITORY:-$DEFAULT_DOCKER_REPOSITORY}
+
+export DOCKER_BUILDKIT=${DOCKER_BUILDKIT:-1}
+
+source $SCRIPTS_DIR/lib/common.sh
+
+check_is_installed docker
+
+# red hat operator certification requires additional labels and metadata
+# in the bundle container image. 
+
+# details on the required bundle labels
+# https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/bundle-directory
+DEFAULT_ADD_RH_CERTIFICATION_LABELS="false"
+ADD_RH_CERTIFICATION_LABELS=${ADD_RH_CERTIFICATION_LABELS:-$DEFAULT_ADD_RH_CERTIFICATION_LABELS}
+
+# details on how versions are reflected by this label.
+# https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/bundle-directory/managing-openshift-versions
+cert_label_openshift_supported_version="com.redhat.openshift.versions"
+DEFAULT_SUPPORTED_OPENSHIFT_VERSIONS="v4.6" # v4.6 and on
```

Comment:
Is this expected to be updated at a certain point?

Reply:
For the time being, "v4.6" indicates that version and later. It can probably stay this way until that changes, or there's another reason to drop support for a particular version. To your question - I expect that this (and other metadata items) may need tweaking over time.